src/diffusers/schedulers/scheduling_ddim.py

```diff
@@ -76,6 +76,41 @@ def alpha_bar(time_step):
     return torch.tensor(betas, dtype=torch.float32)
 
 
+def rescale_zero_terminal_snr(alphas_cumprod):
```

Comment:
Should we refactor this method to take betas as parameters instead of alphas_cumprod?
Because its weird to accept alphas_cumprod and return betas. 
Also, your method comment says betas as the argument.

After all, do we really want an argument that does the rescaling by the scheduler?
Technically, people can always do their betas transformation outside and pass it in as trained_betas.
Also, future schedules may not need this thing anymore.